src/meta-srv/src/metasrv.rs

```diff
@@ -246,6 +248,16 @@ impl MetaSrv {
                 .context(RecoverProcedureSnafu)?;
         }
 
+        if let Some(ref task) = self.greptimedb_telemerty_task {
```

Comment:
It seems that we should only start the task on the meta leader nodes and end the task when the leader stepdown.